### ISSUE TITLE
Getting NullPointerException while trying to run jetty start.run

### ISSUE THREAD

**Issue by DoGi114** [eclipse/jetty.project#4007]
I am trying to make jetty server as service, and im using this guide: https://www.eclipse.org/jetty/documentation/current/startup-windows-service.html However I cant make it work, when trying to proceed with:
`java -jar ..\jetty\start.jar --add-to-start=deploy,http,console-capture`
im getting this:
```
java.lang.NullPointerException
        at sun.nio.fs.WindowsPathParser.parse(Unknown Source)
        at sun.nio.fs.WindowsPathParser.parse(Unknown Source)
        at sun.nio.fs.WindowsPath.parse(Unknown Source)
        at sun.nio.fs.WindowsFileSystem.getPath(Unknown Source)
        at java.nio.file.Paths.get(Unknown Source)
        at org.eclipse.jetty.start.StartArgs.findMavenLocalRepoDir(StartArgs.java:845)
        at org.eclipse.jetty.start.BaseBuilder.<init>(BaseBuilder.java:90)
        at org.eclipse.jetty.start.Main.start(Main.java:439)
        at org.eclipse.jetty.start.Main.main(Main.java:77)
```

**Comment by joakime** [comment]
Link to relevant lines of code that this trips up.

https://github.com/eclipse/jetty.project/blob/84700530e645e812b336747464d6fbbf370c9a20/jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java#L837-L849

Does your windows environment happen to have a `HOME` environment variable setup?
If not, that would be useful to know, as it means we need to adjust this behavior.
Perhaps that code should be using `System.getProperty("user.home")` value instead?

**Comment by DoGi114** [comment]
No it does not.

**Comment by flymoondust** [comment]
I also encome this problem，i wonder why the code couldn't find "maven.local.repo" by function getMavenLocalRepoDir?

public String getMavenLocalRepoDir()
    {
        String localRepo = getProperties().getString("maven.local.repo");

        if (Utils.isBlank(localRepo))
            localRepo = System.getenv("JETTY_MAVEN_LOCAL_REPO");

        if (Utils.isBlank(localRepo))
            localRepo = System.getenv("MAVEN_LOCAL_REPO");

        return localRepo;
    }

**Comment by flymoondust** [comment]
sorry, it's 11 pm here,&nbsp; I'll try tomorrow and reply with result.&nbsp;



---Original---

**Comment by flymoondust** [comment]
first, I don't know if fixed problem of @DoGi114,but i try the zip OK.
second, my scene occurs when compile the source jetty-project of jetty-project 9.4.x, exactly when compile to project jetty-distribution.
But I pull the source to latest, the problem seem gone。
my problem is fixed

**Comment by joakime** [comment]
@flymoondust reports that the simple fix in commit 4d8e8050bd1940e8e1913ec62f6c959a088ff128 works for him, considering this closed.
